web/ui/dashboard/src/app/components/modal/modal.component.html

```diff
@@ -1,5 +1,5 @@
 <div class="fixed bg-black bg-opacity-50 h-screen w-screen top-0 right-0 bottom-0 overflow-hidden z-40" (click)="closeModal.emit()"></div>
-<div class="fixed w-full shadow z-50" [class]="position === 'full' ? 'bg-[#fafafe] ' + classes : 'bg-white-100 ' + classes">
+<div class="fixed w-full shadow z-50" [id]="id" [class]="position === 'full' ? 'bg-[#fafafe] ' + classes : 'bg-white-100 ' + classes">
```

Comment:
What's the use of a dynamic ID? @Oluwadaminiola

Reply:
This is to enable a scroll-to-top effect on the modal